styles.css

```diff
@@ -66,6 +66,7 @@ p {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
+  height: 718px;
```

Comment:
Kindly, I suggest you position the content inside the container through Flexbox, that will ensure that your information is adaptable to all types of devices.